css-display-3/display-contents-text-only-001.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Display: Only text on a display: contents subtree</title>
+<link rel="author" title="Emilio Cobos Álvarez" href="mailto:ecobos@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-display-3/#valdef-display-contents">
+<link rel="match" href="display-contents-text-only-001-ref.html">
+<style>
+  html, body { display: contents; }
+</style>
+PASS
```

Comment:
Hmm, good catch. I've allowed it in my impl, but after double-checking Gecko computes the document element's `display` to `block`. This is not spec'd, but feels sane.

There are tests that specify this, though (which is why I added this test case): https://github.com/w3c/web-platform-tests/blob/master/html/semantics/grouping-content/the-li-element/grouping-li-reftest-no-list-owner.html

Should I:

 * Open a spec issue asking for clarification, or
 * Change my implementation to match Gecko's and add a test for it, or
 * File a Gecko bug, keep this test, and write an additional test for the computed display of the document element?

I'm fine with any of them.